SE ZipFileReader severce platfrom v2.1/MAIN.py

```diff
@@ -0,0 +1,103 @@
+import os, zipfile, pprint, time, datetime, sys
+import MAINCODE_Function_ZFR
+
+class MAIN_ZFR():
+
+
+    # main한 - this code is 한.
+    def main한():
```

Comment:
@choeminjun 
In order to support multi-lingual for this program, it seems you wrote redundant codes such as `main한()` and `mainE()` functions, right?
It isn't a good way because when you try to change some logic in code, you have to change that logic on both functions. So, there are a better way for this purpose, that is, you create a dictionary variable and make it with a key and a value pair. The key points to a message id or name, and the value is a message. And we can create two kind of keys for English and Korean users.

For example, you can define the message variable like this:

```python
i18n = {
  'ask.open.en': 'Do you want to read a zipfile or make a zipfile?',
  'ask.open.ko': 'zip파일을 여시겠습니까 아니면 아카이브 파일을 푸시겠습니까?',
  'cmd.open.en': 'read',
  'cmd.open.ko': '열기',
  ...
}
```

And using above you can replace your code like this:

```python
while True:
  print('zip파일을 여시겠습니까 아니면 아카이브 파일을 푸시겠습니까?')
```

```python
while True:
  print(i18n['ask.open.' + lang])
```

As well as we have to pass the `lang` value from your `main()` function to sub function which you want to run. So, you might better to change main() function like this:

```python
       if languageSV == '한국어' or languageSV == '한':
            print('---환영합니다!---')
            MAIN_ZFR.main('ko')

        elif languageSV == 'English' or languageSV == 'E':
            print('---Hello~!---')
            MAIN_ZFR.main('en')
```

Which means that you use this parameter (en or ko), you can show messages separately for each users who want to see own language.